Tests/TestHelpers/TestHelpers.swift

```diff
@@ -28,12 +67,29 @@ private extension SwiftLintFile {
             .appendingPathComponent("Library")
             .appendingPathComponent("Frameworks")
             .path
-        return [
-            "-F",
-            frameworks,
+
+        let arguments = [
+            "-F", frameworks,
             "-sdk", sdk,
-            "-j4", path!,
+            "-Xfrontend", "-enable-objc-interop",
+            "-j4",
+            path!,
         ]
+#if os(Windows)
+        let XCTestPath = URL(fileURLWithPath: sdk, isDirectory: true)
+            .deletingLastPathComponent()
+            .deletingLastPathComponent()
+            .appendingPathComponent("Library")
+            .appendingPathComponent("XCTest-\(info.defaults.versionXCTest)")
```

Comment:
We could use

```suggestion
            .appendingPathComponent("XCTest-\(SwiftVersion.current.rawValue)")
```

here which directly interacts with SourceKit to retrieve the version. Not sure whether it's safe to assume that Swift version == XCTest version, though. For the 6.2.1 toolchain, that's the case.